lambda_funcs/submission-created.js

```diff
@@ -0,0 +1,39 @@
+// NOTE: This is to test on deploy link. When approved I can change this to our proper URL
```

Comment:
I'm a bit hazy on how this works now, can you provide some background? I thought `Access-Control-Allow-Origin` needed a full URL `https://url.com` to work, not only a string.

Also, does this mean we'd have to set this / remove it any time we want to test this in a preview deploy?

Reply:
Yea, this was just to test on the deploy link. I'm looking into that second question now, and I think the answer is no.